domains/transfer/Transfer.API/API/ApiModels/Requests/RuleExtensions.cs

```diff
@@ -13,4 +14,18 @@ public static class RuleExtensions
         => ruleBuilder
             .LessThan(253402300800)
             .WithMessage("{PropertyName} too high! Please make sure the format is UTC in seconds.");
+
+    public static IRuleBuilderOptions<T, string> MustBeValidBase64<T>(this IRuleBuilder<T, string> ruleBuilder)
+            => ruleBuilder
+                .Must(IsValidBase64String)
+                .WithMessage("Base64-encoded Wallet Deposit Endpoint is not valid");
```

Comment:
This failure message is probably too specific for this rule now - is it possible to assign that message "outside" of the rule?